### PR TITLE
auto upgrade case ocp-13032

### PR DESCRIPTION
**Pull request by zhouying7780**
@geliu2016 @xingxingxia  Please help review , thanks.

https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/60696/console
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/60705/consoleFull

### CODE REVIEW

**Comment by geliu2016** [comment]
LGTM

File: features/upgrade/workloads/oc-upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  # @author yinzhou@redhat.com
+  # @case_id OCP-13032
+  @upgrade-check
+  @users=upuser1,upuser2
+  Scenario: Check some container related oc commands still work after upgrade
+    Given I switch to the first user
+    When I use the "workloads-upgrade" project
+    Given status becomes :running of 1 pods labeled:
+      | name=httpd-example |
+    And evaluation of `pod.name` is stored in the :podname clipboard
```
Comment (xingxingxia) [inline features/upgrade/workloads/oc-upgrade.feature]: this line is not needed

**Comment by xingxingxia** [comment]
@zhouying7780 lgtm except a minor comment

File: features/upgrade/workloads/oc-upgrade.feature (hunk 1 above)
Comment (zhouying7780) [reply features/upgrade/workloads/oc-upgrade.feature]: Thanks, has updated.

**Comment by xingxingxia** [comment]
/lgtm
@pruan-rht @akostadinov , pls help merge, thx

**Pull request merged**